exporters/tests/test_mora_query.py

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright (c) 2017-2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import os
+import csv
+import unittest
+from chardet.universaldetector import UniversalDetector
+import common_queries as cq
+from mora_helpers import MoraHelper
+
+
+class QueryTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.morah = MoraHelper()
```

Comment:
Perhaps it would be better to allow setting the hostname using either configparser, env variables etc. to allow automated testing facilities to specify hostname/port for the running mora instance.